src/app/shared/components/cluster-summary/style.scss

```diff
@@ -69,3 +69,12 @@
     margin-left: -15px;
   }
 }
+
+.km-event-rate-limit {
+  display: flex;
```

Comment:
Flex usage was also mentioned in another comment. Should be replaced with `fxFlex`.